drivers/s3.go

```diff
@@ -372,6 +373,22 @@ func (os *s3Session) saveDataPut(ctx context.Context, name string, data io.Reade
 	return url, nil
 }
 
+func (os *s3Session) DeleteFile(ctx context.Context, name string) error {
```

Comment:
You can consider adding tests unit tests for this part, like this one: https://github.com/livepeer/go-tools/blob/44cbd85855e79c9a8d9327598e0a95d9261c0351/drivers/s3_test.go#L60

I know they are really unit tests, but being pragmatic, they are super helpful to check if everything works fine.

Reply:
Good call, added one and confirmed it works